File: src/Servidor/Transaccional/AccesoDatos/Contrato/IDaoUsuario.cs
using System.Collections.Generic;
using Trascend.Bolet.ObjetosComunes.Entidades;

namespace Trascend.Bolet.AccesoDatos.Contrato
{
    public interface IDaoUsuario : IDaoBase<Usuario, string>
    {
        Usuario Autenticar(Usuario usuario);
    }
}

File: trunk/src/Servidor/Transaccional/AccesoDatos_Fac/Contrato/IDaoUsuario.vb
Imports System.Collections.Generic
Imports Diginsoft.Bolet.ObjetosComunes.Entidades

Namespace Contrato
    Public Interface IDaoUsuario
        Inherits IDaoBase(Of Usuario, String)
        Function Autenticar(ByVal usuario As Usuario) As Usuario
    End Interface
End Namespace